meta/saiserialize.cpp

```diff
@@ -2976,3 +2984,12 @@ void sai_deserialize_queue_attr(
 
     sai_deserialize_enum(s, &sai_metadata_enum_sai_queue_attr_t, (int32_t&)attr);
 }
+
+void sai_deserialize_switch_attr(
```

Comment:
please remove we already have generic function to deserialize attributes for all objects